presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -428,6 +428,38 @@ public void testUnsupportedColumnType()
         session.execute("DROP KEYSPACE keyspace_6");
     }
 
+    public void testNestedCollectionType()
```

Comment:
missing `@Test` annotation

```suggestion
    @Test
    public void testNestedCollectionType()
```